openmc/plots.py

```diff
@@ -401,14 +401,13 @@ def get_plot_xml(self):
         return element
 
 
-class PlotsFile(object):
+class Plots(object):
     """Plots file used for an OpenMC simulation. Corresponds directly to the
```

Comment:
If you agree, consider a similar rewording of the docstrings for `Materials` and `Tallies` as well.